_chapters/chapter_02.md

```diff
@@ -177,9 +202,9 @@ layout: rails_tutorial
       my_favorite_book = Book.new
       ```
 
-      This will create a new instance of `Book` that we can refer to as `my_favorite_book`.
+      This just assigned a new instance of `Book` to a [variable](http://tutorials.jumpstartlab.com/projects/ruby_in_100_minutes.html#2.-variables) so that we can refer to it as `my_favorite_book`.
 
-  1.  Now, give `my_favorite_book` a title. Run:
+  1.  Now, to give this new instance of book a title. Run:
```

Comment:
I *think* I like the old wording better. We've already described that `my_favorite_book` is a new instance of `Book`.

I also like referring to variables in the steps because there's more connection to the things we're working with.